src/wallet/generateFaucetWallet.ts

```diff
@@ -162,6 +210,7 @@ async function getAddressXrpBalance(
     )
     return xrpBalance[0].value
   } catch (err) {
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions -- Is going to be an error
```

Comment:
Since you can `throw` anything in JS, I usually do something like
```ts
} catch (err) {
    if (err instanceOf Error) {
        return `Unable to retrieve ${address} balance. Error: ${err.message}`
    }
    
    throw err
}
```

Reply:
And that should get rid of the `restrict-template-expressions` issue